content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -74,6 +74,16 @@ accessible through the smart button located in the upper-right corner of the for
    :align: center
    :alt: Program items smart button on the loyalty program form.
 
+.. note::
+  In Odoo 17 (and later), when a loyalty/coupon card is related to a contact in the database, a
+  :guilabel:`Loyalty Cards` smart button appears on the contact form.
+
+  .. image:: loyalty_discount/loyalty-cards-smart-button.png
+    :align: center
+    :alt: The Loyalty Card smart button as it appears on a contact form in Odoo 17.
+
+  This smart button **only** appears if a loyaly/coupon card is associated with the contact.
```

Comment:
- fix indentation to align with the block's tag
- separate loyalty card and coupon (they have different meanings despite similar functionality)
- remove redundant sentence

```suggestion
.. note::
   In Odoo 17 (and later), when a loyalty card or coupon is associated with a contact in the
   database, a :guilabel:`Loyalty Cards` smart button conditionally appears on the contact form.

   .. image:: loyalty_discount/loyalty-cards-smart-button.png
      :align: center
      :alt: The Loyalty Card smart button as it appears on a contact form in Odoo 17.
```